### PR TITLE
Add curlylint linter for HTML templates: https://www.curlylint.org

### PR DESCRIPTION
**Pull request by dsully**

### CODE REVIEW

File: lua/mason-registry/curlylint/init.lua [hunk 1]
```diff
@@ -0,0 +1,11 @@
+local Pkg = require "mason-core.package"
+local pip3 = require "mason-core.managers.pip3"
+
+return Pkg.new {
+    name = "curlylint",
+    desc = [[Experimental HTML templates linting for Jinja, Nunjucks, Django templates, Twig, Liquid]],
+    homepage = "https://www.curlylint.org/",
+    languages = { Pkg.Lang.Python },
```
Comment (williamboman) [inline lua/mason-registry/curlylint/init.lua:8]: For some tools the languages to include can be a bit awkward, but I think the following would be most correct:

```suggestion
    languages = { Pkg.Lang.Django, Pkg.Lang.Ninja, Pkg.Lang.Nunjucks, Pkg.Lang.Twig, Pkg.Lang.Liquid },
```

**Review by williamboman — APPROVED** [review]
Cool!

**Comment by williamboman** [comment]
@williambotman /fixup

**Comment by dsully** [comment]
Thanks for the fixups!

**Comment by williamboman** [comment]
I just realized I accidentally wrote Ninja 🥷, will fix

**Pull request merged**